components/automate-gateway/handler/ssoconfig.go

```diff
@@ -72,6 +69,52 @@ func (a *SsoConfig) GetSsoConfig(ctx context.Context, in *empty.Empty) (*sso.Get
 	}, nil
 }
 
+func (a *SsoConfig) DeleteSsoConfig(ctx context.Context, in *empty.Empty) (*sso.DeleteSsoConfigResponse, error) {
+	err := a.validateDeploymentType(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Printf("Received request to delete sso config")
+	res, err := a.getConfigData(ctx)
+
+	if err !=nil {
+		return nil, err
+	}
+	
+	if res.Config.Dex != nil {
+		url, err := getBastionUrl()
+		if err != nil {
+			return nil, err
+		}
+		fileName := "revert-status.txt"
+		err = ioutil.WriteFile(ssoFilesPath+fileName, []byte("Pending"), 0777)
```

Comment:
As I see this is a file which will save the state of the Revert Task.
Which means this file will be checked for status later, to know if the task is finished or not.

But here is my question, if check status is going to be called from UI via API, then if we had more than 1 Automate Node, then the status only going to be available in only one Automate Node, not Both (assuming this file is only the source of the revert task state).

This makes me wonder if this design had considered the Multiple Node of Automate and how state will be maintained across them. If this is considered in the design please highlight it.

@sandhi18  @karunesh-tech  ^